### DISCUSSION TITLE
Nested bandwidth limit with both limits as classic and refill intervally

### DISCUSSION THREAD

**Discussion by abhayJava** [vladimir-bukhtoyarov/bucket4j#262] [Q&A]
Hi, 

I have throttle condition which has initial capacity as 500, refill rate at 100 tokens per minute intervally , but at the same time i want to restrict requests to 10 requests per second

So, I added bandwidth limit for refill at 100 tokens/minute using classic and refill as intervally,  and then want to add 2nd level limit as 10 request/second, but dont want to refill greedily , so planning to use classic, but what should be initial capacity and refill rate for 2nd limit as i want refill to be driven by 1st limit only i.e. 100 tokens/minute

```java
 private fun createNewBucket():Bucket{
        var overDraft: Long = 500
        var refill:Refill  = Refill.intervally(100,Duration.ofSeconds(60))
        var limit:Bandwidth  = Bandwidth.classic(overDraft,refill)
        return Bucket.builder().addLimit(limit).addLimit(Bandwidth.simple(10, Duration.ofSeconds(1)))
                .build()

    }
```
    Thanks,
Abhay

**Comment by vladimir-bukhtoyarov** [comment]
Each bandwidth do refilling independently from each other. It is not possible to share the refill between bandwidths.
